tools/ci_connector_ops/ci_connector_ops/pipelines/commands/airbyte_ci.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import click
+from .metadata_service import metadata_service
+from .connectors_ci import connectors_ci
+from ci_connector_ops.pipelines.contexts import CIContext
+from ci_connector_ops.pipelines.utils import (
+    get_current_epoch_time,
+    get_current_git_branch,
+    get_current_git_revision,
+    get_modified_files,
+)
+
+CI_CONTEXT_VALUES = [member.value for member in CIContext]
+
+
+@click.group(help="Airbyte CI top-level command group.")
+@click.option("--is-local/--is-ci", default=True)
+@click.option("--git-branch", default=get_current_git_branch, envvar="CI_GIT_BRANCH")
+@click.option("--git-revision", default=get_current_git_revision, envvar="CI_GIT_REVISION")
+@click.option(
+    "--diffed-branch",
+    help="Branch to which the git diff will happen to detect new or modified connectors",
+    default="origin/master",
+    type=str,
+)
+@click.option("--gha-workflow-run-id", help="[CI Only] The run id of the GitHub action workflow", default=None, type=str)
+@click.option("--ci-context", default="manual", envvar="CI_CONTEXT", type=click.Choice(CI_CONTEXT_VALUES))
```

Comment:
I think you can directly use the enum class in `Choice`. It worked for ConnectorLanguage:
https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/connectors_ci.py#L137